packages/docs/pages/checkbox.tsx

```diff
@@ -134,6 +134,51 @@ const CheckboxPage = () => {
                 </Fragment>
               ),
             },
+            {
+              id: 'badge',
+              title: 'Badge',
+              render: () => (
+                <Fragment key="badge">
+                  <Text>
+                    <Code primary>Checkboxes</Code> support <Code primary>Badge</Code> to display{' '}
+                    specify information
```

Comment:
Do you mean `specific information`? 

I also wonder if this fits well 🤔.  My take is that we can change this to `status information`.

Reply:
@jorgemoya yeap, it was a typo. i mean `specific information`
But your version would be better
I have replaced with `status information`.